java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsFileAccess.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Android WebSettings file access
+ * @kind problem
+ * @description Enabling access to the file system in a WebView can enable access to sensitive information.
+ * @id java/android-websettings-file-access
+ * @problem.severity warning
+ * @security-severity 6.5
+ * @precision high
```

Comment:
Since this won't always be a security problem, let's get this down to medium so that the query becomes opt-in for users that want to see more results.

```suggestion
 * @precision medium
```